.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "CurrentProjectSetting": null
+}
```

Comment:
Can you please remove the visual studio files from the patch?

Reply:
Yeah I will resolve the errors and issues